dev-util/bear/bear-3.0.10.ebuild

```diff
@@ -0,0 +1,75 @@
+# Copyright 2020-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="Build EAR generates a compilation database for clang tooling"
+HOMEPAGE="https://github.com/rizsotto/Bear"
+SRC_URI="https://github.com/rizsotto/Bear/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc64"
+IUSE="test"
+
+RDEPEND="
+>=net-libs/grpc-1.26
+>=dev-libs/libfmt-6.2
+>=dev-libs/spdlog-1.5
+>=dev-cpp/nlohmann_json-3.7
+>=dev-db/sqlite-3.14
+"
+
+DEPEND="${RDEPEND}
+test? (
+	>=dev-cpp/gtest-1.10
+)"
+
+BDEPEND="test? (
+	app-shells/bash
+	dev-python/lit
```

Comment:
If this test is still valid you can't drop PYTHON_COMPAT nor python-any-r1

Reply:
Also you don't really need to add bash sad requirement as it is a must have in Gentoo

Reply:
OH, yeah that makes sense - ty